packages/table-core/src/core/column.ts

```diff
@@ -52,12 +52,16 @@ export function createColumn<TData extends RowData, TValue>(
         let result = originalRow as Record<string, any>
 
         for (const key of accessorKey.split('.')) {
-          result = result[key]
-          if (process.env.NODE_ENV !== 'production' && result === undefined) {
-            throw new Error(
-              `"${key}" in deeply nested key "${accessorKey}" returned undefined.`
-            )
+          if (result == null || !(key in result)) {
```

Comment:
should this be "===" triple equals equality check? and actually, wouldn't we want to check for undefined? Why is the result assignment moved to the last line?

Reply:
I used "==" to handle both undefined and null, because you cannot use the ```in``` operator on either of them. Unless you want to handle undefined and null differently?

Reply:
@SeanStove I'm working on a PR where I'm simply making this change. Wonder if it works in your testing

```ts
accessorFn = (originalRow: TData) => {
        let result = originalRow as Record<string, any>

        for (const key of accessorKey.split('.')) {
          result = result?.[key]
          if (process.env.NODE_ENV !== 'production' && result === undefined) {
            console.warn(
              `"${key}" in deeply nested key "${accessorKey}" returned undefined.`
            )
          }
        }

        return result
      }
```

Reply:
@KevinVandy Tested and it covers the 3 cases I described above. Thanks for this.

Reply:
Shipped this last week if you hadn't seen that yet